drivers/hmis/lib/ce_workflows/ac_builder.rb

```diff
@@ -0,0 +1,381 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+##
+# CeWorkflows::AcBuilder
+# Utility for building Coordinated Entry (CE) workflow definitions specific to the AC implementation.
+# Intended for use in scripts and Rake tasks that automate the setup or teardown of AC CE workflows.
+# Not intended for use in production application logic.
+module CeWorkflows
+  class AcBuilder
+    CE_STEP_FORMS = {
+      initial_review: 'housing_workflow_initial_review',
+      initial_client_engagement: 'housing_workflow_initial_client_engagement',
+      client_engagement: 'housing_workflow_client_engagement',
+      client_offer_outcome: 'housing_workflow_client_offer_outcome',
+      provider_outcome_1: 'housing_workflow_provider_outcome_1',
+      provider_outcome_2: 'housing_workflow_provider_outcome_2',
+      provider_outcome_3: 'housing_workflow_provider_outcome_3',
```

Comment:
decided to do separate forms for each iteration of the provider outcome task, even though forms are identical. we could probably use the same form if we want to. i'm unclear on benefits/pitfalls.

Reply:
I think I'm a bit worried about the reporting implication of re-using the same CustomDataElementDefinition (ie `housing_workflow_provider_outcome_note`) across multiple steps. (If we used the same form for all 3 steps, then all 3 steps would be collecting CDEs using the same custom data element definition). That might be fine––each CDE will be "owned" by a different node instance–– but we'd need to figure out how to solve that for reporting. It feels like it will be more straightforward to use unique CDEDs (`housing_workflow_provider_outcome_1_note`, `housing_workflow_provider_outcome_2_note`) to more easily distinguish where they were collected.

I definitely feel like we _should_ be able to support either way, but this felt safer to me.